pkg/node/manager.go

```diff
@@ -300,6 +300,7 @@ func DeleteNode(ni Identity, routesTypes RouteType) {
 
 			deleteTunnelMapping(n.IPv4AllocCIDR)
 			deleteTunnelMapping(n.IPv6AllocCIDR)
+			deleteIPRoute(n)
```

Comment:
Should the routes be removed before the tunnel? I'm not sure which which leads to which.